specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/consumption.json

```diff
@@ -280,6 +280,14 @@
           }
         },
         "parameters": [
+          {
+            "name": "scope",
+            "description": "The scope of the budgets. The scope can be 'subscriptions/{subscriptionId}' for a subscription, or 'subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}' for a resource group under a subscription.",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true
+          },
           {
             "$ref": "#/parameters/apiVersionParameter"
           },
```

Comment:
{
             "$ref": "#/parameters/subscriptionIdParameter"		         
}

Is not required in the parameters here as its not expliciluty declared in the path.

same case on all such budgets paths. Please check. For example, check usage details operation

Reply:
👍